.local/render-allmetrics.py

```
@@ -0,0 +1,210 @@
+import matplotlib.pyplot as plt
+
+DIR = sys.argv[1] if len(sys.argv) > 1 else sorted(glob.glob('/tmp/proto-sweep-2*'))[-1]
+print(f"rendering: {DIR}")
+
+CH_URL = 'http://localhost:30123'
+CH_AUTH = ('pixie', 'pixie_password')
+
+def ch_query(q):
+    try:
+        url = f"{CH_URL}/?query=" + urllib.parse.quote(q + " FORMAT TabSeparated")
+        out = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)